types/react-table/react-table-tests.tsx

```diff
@@ -28,7 +28,7 @@ const columns: Array<Column<Data>> = [
         accessor: (data: Data) => data.age,
         Cell: props => <span className='number'>{props.value}</span>
       },
-      { Header: "Status", accessor: "status" }
+      { Header: "Status", accessor: "progress" }
```

Comment:
The updated typedefs found this error. `status` does not exist in interface `Data`.

Reply:
How about changing the header name to `Progress`, so that it matches the `accessor`? Having the header differ from the accessor may cause confusion for developers modifying the tests in the future 😄

```suggestion
      { Header: "Progress", accessor: "progress" }
```